shark/iree_utils.py

```diff
@@ -152,6 +159,15 @@ def get_iree_compiled_module(module,
 
     return get_iree_module(module, device, input_type, args, func_name)
 
+def export_iree_module_to_vmfb(module, device: str, directory: str):
```

Comment:
Nit:Shark module? In our code we can call IREE SHARK

Reply:
There's actually a bunch of these in iree_utils, I'll push this up in a separate patch! :)